flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
why no `aarch64` ?

Reply:
The company currently does not have this architecture.

Reply:
You're building from source, if it builds and runs you can remove the restriction.

Reply:
Fix